bodhi/server/util.py

```diff
@@ -604,7 +604,10 @@ def cmd(cmd, cwd=None):
     if out:
         log.debug(out)
     if err:
-        log.error(err)
+        if p.returncode == 0:
+            log.info(err)
+        else:
+            log.error(err)
```

Comment:
This change looks good, but we will want to add some tests to make sure it works correctly (and stays working correctly).

I suggest using [```mock```](http://www.voidspace.org.uk/python/mock/index.html) to fake a call to subprocess so you can create conditions for a few tests:

0. There is no err output.
1. There is err output and the exit code is 0.
2. There is err output and the exit code is not 0.

Let me know if you need help getting started with these!